examples/v1beta1/pipelineruns/pipelinerun-results-with-params.yml

```diff
@@ -0,0 +1,36 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: test-case
+  name: test-case-run
+spec:
+  params:
+  - name: prefix
+    value: prefix
+  pipelineSpec:
+    params:
+    - name: prefix
+    tasks:
+    - name: generate-suffix
+      taskSpec:
+        results:
+        - name: suffix
+        steps:
+        - name: generate-suffix
+          image: alpine
+          script: |
+            echo -n "suffix" > $(results.suffix.path)
+    - name: do-something
+      runAfter:
+      - generate-suffix
```

Comment:
why is there a runAfter here? i thought the intention was for the variable interpolation to imply the ordering?

Reply:
You are right. This is the example from the original issue.